### PR TITLE
Fix some low-hanging fruit in event subpage instructions.

### PR DESCRIPTION
**Pull request by jpmedley**
This is not an exhaustive update of the page. The link for the example event was to a deprecated page. In the process of updating that, I looked for other low hanging fruit. I hope you'll accept it in the spirit it was intended: not wanting perfect to be the enemy of good.

### CODE REVIEW

**Comment by jpmedley** [comment]
By the way, because we've hidden compat tables for keys not present in the database, the example table on this page does not show up. https://developer.mozilla.org/en-US/docs/MDN/Structures/Page_types/API_event_subpage_template

File: files/en-us/mdn/structures/page_types/api_event_subpage_template/index.html [hunk 1]
```diff
@@ -116,7 +115,7 @@ <h2 id="Specifications">Specifications</h2>
  </tbody>
 </table>
 
-<h2 id="Browser_compatibility_2">Browser compatibility</h2>
+<h2 id="Browser_compatibility">Browser compatibility</h2>
```
Comment (nschonni) [inline files/en-us/mdn/structures/page_types/api_event_subpage_template/index.html:118]: This ID already exists on 42, which is why it has the number
```suggestion
<h2 id="Browser_compatibility_2">Browser compatibility</h2>
```

File: files/en-us/mdn/structures/page_types/api_event_subpage_template/index.html [hunk 2]
```diff
@@ -16,29 +16,28 @@ <h2 id="Remove_this_section_before_publishing">Remove this section before publis
...
+  <div>\{{APIRef("<em>GroupDataName</em>")}} — this generates the left hand reference sidebar showing quick reference links related to the current page. For example, every page in the <a href="/en-US/docs/Web/API/WebVR_API">WebVR API</a> has the same sidebar, which points to the other pages in the API. To generate the correct sidebar for your API, you need to add a GroupData entry to the <a href="https://github.com/mdn/yari/blob/main/kumascript/macros/GroupData.json">GroupData.jjson</a> file in our Yari repo, and include the entry's name inside the macro call in place of <em>GroupDataName</em>. See our <a href="/en-US/docs/MDN/Contribute/Howto/Write_an_API_reference/Sidebars">API reference sidebars</a> guide for information on how to do this.</div>
```
Comment (chrisdavidmills) [inline files/en-us/mdn/structures/page_types/api_event_subpage_template/index.html:32]: "jjson" -> "json"

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Some nice updates here, thanks @jpmedley . Just one comment from me to address, and one from @nschonni , and we should be done.

File: files/en-us/mdn/structures/page_types/api_event_subpage_template/index.html (hunk 1 above)
Comment (jpmedley) [reply files/en-us/mdn/structures/page_types/api_event_subpage_template/index.html:118]: If a contributor cuts and pasts this page, they would need to change this ID. To eliminate that requirement, I've changed the ID of the heading in line 42 instead.

File: files/en-us/mdn/structures/page_types/api_event_subpage_template/index.html (hunk 2 above)
Comment (jpmedley) [reply files/en-us/mdn/structures/page_types/api_event_subpage_template/index.html:32]: Fixed.

**Review by sideshowbarker — APPROVED** [review]

**Review by chrisdavidmills — APPROVED** [review]
LGTM!

**Pull request merged**